LogicLayer/LogicLayer.csproj

```diff
@@ -17,7 +17,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="JetBrains.Annotations" Version="2019.1.1" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
```

Comment:
Git blame tells me `Wiktor: Added xunit to Logic Layer (temporarily)`. Time for that temporarily to end.

Explains why the project icon in VS turned to a test project, I thought it was a VS bug :P

Reply:
Did it XD I have completely forgotten that I added it to LogicLayer once.